examples/bri-3/src/bri/communication/agents/bpiMessages.agent.ts

```diff
@@ -10,6 +10,7 @@ import {
   BPI_MESSAGE_ALREADY_EXISTS,
   NOT_FOUND_ERR_MESSAGE,
 } from '../api/err.messages';
+import { NOT_FOUND_ERR_MESSAGE as BPI_SUBJECT_NOT_FOUND_ERR_MESSAGE } from '../../identity/bpiSubjects/api/err.messages';
```

Comment:
why do we need this conversion? Are these not static strings?

Reply:
because same import 2 lines above